src/app/components/views/thumbnail/preview.component.ts

```diff
@@ -88,7 +97,9 @@ export class PreviewComponent implements OnInit, OnDestroy {
       clearInterval(this.scrollInterval);
     }
 
-    if (this.returnToFirstScreenshot) {
+    if (this.video.defaultScreen) {
```

Comment:
Should this not also take into account `this.returnToFirstScreenshot`?

Reply:
Good catch -- I'll look into it and make another commit tomorrow (I'm away today) 🤝

Reply:
Thank you Cal 🙇‍♂️ -- I think it's fixed 👍